lib/Parse/ParseDecl.cpp

```diff
@@ -5759,7 +5759,11 @@ ParserStatus Parser::parseGetSet(ParseDeclOptions Flags,
 
       // parsingLimitedSyntax mode cannot have a body.
       if (parsingLimitedSyntax) {
-        diagnose(Tok, diag::expected_getset_in_protocol);
+        // No error should be generated on implicitly created getter.
+        auto IsImplicitGetter = Kind == AccessorKind::Get && Loc.isInvalid();
```

Comment:
I believe this is always `true` since this is in `NotAccessor = parseAccessorIntroducer()` branch.

Reply:
Thanks! I removed unnecessary `IsImplicitGetter` and added a test case for implicit getter :)